.eslintrc.json

```diff
@@ -29,6 +29,8 @@
     ],
     "@typescript-eslint/no-unused-vars": "off",
     "@typescript-eslint/explicit-function-return-type": "off",
+    "@typescript-eslint/restrict-plus-operands": "off",
+    "@typescript-eslint/strict-boolean-expressions": "off",
```

Comment:
IMO the relatively recent `?.` and preferring `x == null` and `x != null` now provides a nice balance of ergonomics / explicit code, e.g., `if (x?.length > 0)` and `if (x?.f?.x != null)`